charon/tests/crate_data.rs

```diff
@@ -1,11 +1,11 @@
 #![feature(box_patterns)]
 
+use charon_lib::llbc_ast::*;
+use charon_lib::values::Literal::Scalar;
```

Comment:
Once again, please only import enum variants that are used throughout the file. You can move this import to the function where you use them.